shared/js/abp-preprocessed.es6.js

```diff
@@ -140,3 +147,41 @@ function getVersionParam () {
 
     return versionParam
 }
+
+function fetchSurrogateCode () {
+    let trackersSurrogateListEtag = settings.getSetting('trackersSurrogateList-etag') || ''
+    // reset etag to get a new list copy if we don't have data
+    if (!trackersSurrogateList || !trackersSurrogateListEtag) trackersSurrogateListEtag = ''
+    // load surrogates list
+    load.loadExtensionFile({
+        url: constants.trackersSurrogateList,
+        etag: trackersSurrogateListEtag,
+        source: 'external'
+    }, (listData, response) => {
+        const newTrackersSurrogateListEtag = response.getResponseHeader('etag') || ''
+        const clientChecksum = crypto.createHash('md5').update(listData).digest('base64')
+
+        settings.updateSetting('trackersSurrogateList-etag', newTrackersSurrogateListEtag);
+        //  check that etag hash matches hash of file received
+        if (!clientChecksum || clientChecksum.substring(0,6) !== newTrackersSurrogateListEtag) {
+            console.log("Checksum didn't match")
```

Comment:
Dumb niggle but could you specify that the trackers surrogate checksum didnt match and use single quotes.

Reply:
I think we should ditch the crypto checksum at this time. this is overcomplicating it at this point.